OrbitCore/ElfFile.cpp

```diff
@@ -81,14 +92,70 @@ bool ElfFile::GetFunctions(Pdb* pdb, std::vector<Function>* functions) const {
     }
 
     // Limit list of symbols to functions. Ignore sections and variables.
-    if (symbol_ref->getType().get() != llvm::object::SymbolRef::ST_Function) {
+    if (symbol_ref.getType().get() != llvm::object::SymbolRef::ST_Function) {
       continue;
     }
 
     functions->emplace_back(name, Path::GetFileName(file_path_),
-                            symbol_ref->getValue(), symbol_ref->getSize(), pdb);
+                            symbol_ref.getValue(), symbol_ref.getSize(), pdb);
     function_added = true;
   }
 
   return function_added;
 }
+
+template <typename ElfT>
+uint64_t ElfFileImpl<ElfT>::GetLoadBias() const {
+  const llvm::object::ELFFile<ElfT>* elf_file = object_file_->getELFFile();
+
+  uint64_t min_vaddr = UINT64_MAX;
+  llvm::Expected<typename ElfT::PhdrRange> range = elf_file->program_headers();
+
+  if (!range) {
+    FATAL(absl::StrFormat("No program headers found in %s\n", file_path_));
```

Comment:
We shouldn't kill Orbit if we don't find the proper header for an elf file...  Could we simply return UINT64_MAX as an error code?

Reply:
Are there any elf files that don't have program headers? They are needed for execution. All the symbols only files that I came checked, also had program headers.

Reply:
an elf-file without program headers is pretty much not-loadable, but it theory an object file is one example. we should not encounter these here.

Reply:
made a return value optional and added a test for it.